### PR TITLE
Color code VUS based on timestamp

### PR DESCRIPTION
**Pull request by calvinlu3**
This is related to https://github.com/oncokb/oncokb/issues/3674

image

### CODE REVIEW

File: src/main/webapp/app/shared/icons/InfoIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          <span
+            className={classNames('fa-layers fa-fw', props.className)}
+            style={{
+              fontSize: '1rem',
+              ...style,
+            }}
+          >
+            <FontAwesomeIcon icon={farCircle} />
+            <FontAwesomeIcon icon={faInfo} transform="shrink-9" />
+          </span>
```
Comment (zhx828) [inline src/main/webapp/app/shared/icons/InfoIcon.tsx:36]: We should generic the logic combing this one and the one in ActionIcon.

**Review by zhx828 — CHANGES_REQUESTED** [review]
One minor request.

**Comment by calvinlu3** [comment]
Increased size of badge
image

**Review by zhx828 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**